### PR TITLE
Camlzip 1.13

### PR DESCRIPTION
**Pull request by xavierleroy**
- https://github.com/xavierleroy/camlzip/pull/44
- https://github.com/xavierleroy/camlzip/issues/46
- https://github.com/xavierleroy/camlzip/pull/47
- https://github.com/xavierleroy/camlzip/pull/48

### CODE REVIEW

File: packages/camlzip/camlzip.1.13/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis:
+  "Accessing compressed files in ZIP, GZIP and JAR format."
```
Comment (mseri) [inline packages/camlzip/camlzip.1.13/opam:3]:
```suggestion
  "Accessing compressed files in ZIP, GZIP and JAR format"
```

File: packages/camlzip/camlzip.1.13/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description:
+  "The Camlzip library provides easy access to compressed files in ZIP and GZIP format, as well as to Java JAR files.  It provides functions for reading from and writing to compressed files in these formats."
+maintainer: ["Xavier Leroy <xavier.leroy@college-de-france.fr>"]
+authors: ["Xavier Leroy"]
+homepage: "https://github.com/xavierleroy/camlzip"
+bug-reports: "https://github.com/xavierleroy/camlzip/issues"
+dev-repo: "git+https://github.com/xavierleroy/camlzip.git"
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
```
Comment (mseri) [inline packages/camlzip/camlzip.1.13/opam:11]:
```suggestion
license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
```

**Comment by xavierleroy** [comment]
Sorry for the warnings in the opam file, I was fixing them as you were reviewing.

**Comment by mseri** [comment]
No problem! Thanks for the fast response

**Comment by mseri** [comment]
Thanks

**Pull request merged**